programs/axelar-solana-gateway/src/processor/initialize_payload_verification_session.rs

```diff
@@ -77,9 +83,36 @@ impl Processor {
             GatewayConfig::read(&data).ok_or(GatewayError::BytemuckDataLenInvalid)?;
         assert_valid_gateway_root_pda(gateway_config.bump, gateway_root_pda.key)?;
 
+        // Check: Signing verifier set is valid and sufficiently recent
```

Comment:
Before, signature verification session PDAs were permissionless, anyone could create them for any arbitrary verifier set hash. 

But now we do enforce that the aspiring verifier set hash to be already known/trusted by the Gateway.